util/MySqlMigrations/Migrations/DatabaseContextModelSnapshot.cs

```diff
@@ -490,7 +490,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<int>("Id")
                         .ValueGeneratedOnAdd()
                         .HasColumnType("int")
-                        .HasAnnotation("MySql:ValueGenerationStrategy", MySqlValueGenerationStrategy.IdentityColumn);
+                        .HasAnnotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.IdentityByDefaultColumn);
```

Comment:
@withinfocus It looks like these manual edits will get reverted every time we run EF migrations?

Steps to reproduce on main. 

```
cd dev
pwsh ef_migrate.ps1 Test
git diff
```


![image](https://github.com/bitwarden/server/assets/43214426/b1676449-7bb6-4c71-9484-a9079d1ac478)



Is the expectation to revert what the tools do every time?

Reply:
Sigh, I kinda expected that as we tried to debug some issues with the EF migration process. They aren't affecting anything anyway so this is fine and despite the bad Git history can merge here.